actors/miner/tests/precommitted_sector_stores.rs

```diff
@@ -0,0 +1,111 @@
+use cid::Cid;
+use fil_actor_miner::State;
+use fil_actor_miner::{SectorPreCommitInfo, SectorPreCommitOnChainInfo};
+use fil_actors_runtime::test_utils::make_sealed_cid;
+use fil_actors_runtime::test_utils::MockRuntime;
+use fil_actors_runtime::DealWeight;
+use fvm_shared::bigint::Zero;
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::sector::RegisteredSealProof;
+use fvm_shared::sector::SectorNumber;
+
+mod util;
+use util::*;
+
+#[test]
+fn put_get_and_delete() {
+    let h = ActorHarness::new(0);
```

Comment:
Doesn't seem to be biting you here but the go test uses a different harness for state tests. You don't really need the harness methods for error handling because rust is less annoying. But for consistency I think we should use a new harness in order to duplicate the state harness construction code.  This should also help with easy setting of period boundary for later tests.

Reply:
Done.